{"name": "登录成功", "status": "skipped", "statusDetails": {"message": "Skipped: 重复用例，隐藏不运行", "trace": "('D:\\\\ui\\\\test_case\\\\test_testin.py', 25, 'Skipped: 重复用例，隐藏不运行')"}, "description": "测试登录模块", "parameters": [{"name": "case", "value": "[3, '错误的手机号、正确的密码，登录失败', 13715297701, 'Aa12345678', 'y', None, '用户名或密码错误']"}], "start": 1715047700447, "stop": 1715047700447, "uuid": "3e0b301e-8931-441d-bbff-cc1b35111a68", "historyId": "37508103aba5bdb5a459d67105acd870", "testCaseId": "7d0eb7ff4cf72430f89c3a6b1ae19d96", "fullName": "test_case.test_testin.TestTestIn#test_01_login", "labels": [{"name": "feature", "value": "login"}, {"name": "epic", "value": "项目hkc"}, {"name": "tag", "value": "@pytest.mark.skip(reason='重复用例，隐藏不运行')"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_testin"}, {"name": "subSuite", "value": "TestTestIn"}, {"name": "host", "value": "hkc0208725"}, {"name": "thread", "value": "43120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_testin"}]}